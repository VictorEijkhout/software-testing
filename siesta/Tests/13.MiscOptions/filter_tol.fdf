%include ../base_wat_smallbox.fdf
SystemLabel     filter_tol

%block ChemicalSpeciesLabel
 1  8  O
 2  1  H
%endblock ChemicalSpeciesLabel

FilterTol    0.008 Ry
MeshCutoff   200.0 Ry

