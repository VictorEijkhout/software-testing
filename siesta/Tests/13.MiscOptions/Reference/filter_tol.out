Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/13.MiscOptions/work/filter_tol
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:40:26

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.20348
************************** Dump of input data file ****************************
%include ../../basejob_2.fdf
SystemLabel     filter_tol
%block ChemicalSpeciesLabel
 1  8  O
 2  1  H
%endblock ChemicalSpeciesLabel
FilterTol    0.008 Ry
MeshCutoff   200.0 Ry
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 2 Water molecules
reinit: -----------------------------------------------------------------------
reinit: System Label: filter_tol
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    8 Label: O
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: O
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/O.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf

---- Pseudopotential check for O 

Pseudized shells:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
Valence configuration for ps generation: (assumed as above)

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  15.999        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Filter: kinetic energy tolerance:    0.008000 Ry
Using PAO.SplitTailNorm
atom: Called for O                     (Z =   8)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1366
V l=1 = -2*Zval/r beyond r=  1.1332
V l=2 = -2*Zval/r beyond r=  1.1298
V l=3 = -2*Zval/r beyond r=  1.1264
All V_l potentials equal beyond r=  1.1355
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1366
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.667 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     76.727 Ry
atom: Maximum radius for chlocal:    1.37241
atom: Maximum radius for r*vlocal+2*Zval:    1.25469

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.293209   el= -1.742414   Ekb=  9.076872   kbcos=  0.325320
GHOST: No ghost state for L =  1
   l= 1   rc=  1.284119   el= -0.676589   Ekb= -8.194485   kbcos= -0.396010
GHOST: No ghost state for L =  2
   l= 2   rc=  1.433152   el=  0.013307   Ekb= -2.071665   kbcos= -0.003690
GHOST: No ghost state for L =  3
   l= 3   rc=  1.540891   el=  0.019584   Ekb= -0.817137   kbcos= -0.000371

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    3.560943
                 energy =   -1.731321
                kinetic =    1.587329
    potential(screened) =   -3.318651
       potential(ionic) =  -11.260325
PAO Filter: Cutoff estimated from residual kin energy:   41.05 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 25  6

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.290924
                 energy =   -0.665565
                kinetic =    4.934810
    potential(screened) =   -5.600376
       potential(ionic) =  -13.346429
PAO Filter: Cutoff estimated from residual kin energy:   75.72 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 34 10

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    4.290924
                 energy =    2.236783
                kinetic =    4.278617
    potential(screened) =   -2.041834
       potential(ionic) =   -8.171412
PAO Filter: Cutoff estimated from residual kin energy:   84.20 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 36 10
atom: Total number of Sankey-type orbitals:  9
Maximum filter cutoff from orbitals:    84.1991 Ry

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.290924
VNA Filter: diagnostic kinetic energy tol:    0.003000 Ry
VNA Filter: Nominal cutoff (before filtering):   43.86 Ry
VNA Filter: Cutoff used:   171.83 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 47 16

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=1  polorb=1  (1s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=0  polorb=0  (2p)  (perturbative polarization orbital) (from 1s)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Filter: kinetic energy tolerance:    0.008000 Ry
Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.285354
                 energy =   -0.456120
                kinetic =    0.900445
    potential(screened) =   -1.356565
       potential(ionic) =   -1.890080
PAO Filter: Cutoff estimated from residual kin energy:   21.28 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 26  6

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s
Split based on tail norm

   izeta = 1
                     rc =    5.285354
                 energy =    0.642323
                kinetic =    1.282472
    potential(screened) =   -0.640149
       potential(ionic) =   -1.117275
PAO Filter: Cutoff estimated from residual kin energy:   25.52 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 27  6
atom: Total number of Sankey-type orbitals:  4
Maximum filter cutoff from orbitals:    84.1991 Ry

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.285354
VNA Filter: diagnostic kinetic energy tol:    0.003000 Ry
VNA Filter: Nominal cutoff (before filtering):   33.11 Ry
VNA Filter: Cutoff used:   171.83 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 55 20

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    8 O                       # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
O                     2                    # Species label, number of l-shells
 n=2   0   1                         # n, l, Nzeta 
   3.561   
   1.000   
 n=2   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.291   
   1.000   
H                     1                    # Species label, number of l-shells
 n=1   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.285   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file O.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      1.43052   1.10738   0.00000  2        2
siesta:     -1.43052   1.10738   0.00000  2        3
siesta:      0.00000   9.44863   0.00000  1        4
siesta:      1.43052   8.34125   0.00000  2        5
siesta:     -1.43052   8.34125   0.00000  2        6

siesta: System type = slab      

initatomlists: Number of atoms, orbitals, and projectors:      6    34    68

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   200.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.000000
Total ionic charge:    16.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        3.000000    0.000000    0.000000
        0.000000   10.000000    0.000000
        0.000000    0.000000    5.000000

outcell: Cell vector modules (Ang)   :    3.000000   10.000000    5.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    150.0000
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 7>
  <dData1D:(new from dSpData1D) n=994, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 8>
  <dData2D:DM n=994 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

InitMesh: MESH = 30 x 90 x 48 = 129600
InitMesh: Mesh cutoff (required, used) =   200.000   223.865 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -194.812793
siesta: Eions   =      1621.026762
siesta: Ena     =       347.676395
siesta: Ekin    =       736.417290
siesta: Enl     =      -133.492932
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -27.859978
siesta: DUscf   =         4.830077
siesta: DUext   =         0.000000
siesta: Ex      =      -204.963024
siesta: Ec      =       -25.388295
siesta: Exc     =      -230.351320
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -915.396118
siesta: Etot    =      -923.807230
siesta: FreeEng =      -923.807230

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -915.396118     -923.807230     -923.807230  0.623394 -3.028164  4.611133
timer: Routine,Calls,Time,% = IterSCF        1       0.036   4.70
   scf:    2     -926.639064     -925.518565     -925.518565  0.130067 -5.496414  1.406817
   scf:    3     -925.751132     -925.706943     -925.706943  0.065000 -5.308112  0.263321
   scf:    4     -925.713905     -925.710877     -925.710877  0.005194 -5.363257  0.110576
   scf:    5     -925.714488     -925.712752     -925.712752  0.003596 -5.405571  0.064070
   scf:    6     -925.714242     -925.713529     -925.713529  0.002934 -5.435886  0.026740
   scf:    7     -925.713663     -925.713604     -925.713604  0.000669 -5.438691  0.012013
   scf:    8     -925.713630     -925.713617     -925.713617  0.000347 -5.438147  0.004873
   scf:    9     -925.713621     -925.713620     -925.713620  0.000120 -5.436964  0.002708
   scf:   10     -925.713620     -925.713620     -925.713620  0.000046 -5.436061  0.001661
   scf:   11     -925.713621     -925.713620     -925.713620  0.000042 -5.435522  0.001146
   scf:   12     -925.713621     -925.713620     -925.713620  0.000035 -5.435173  0.000802

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000354369
max |H_out - H_in|      (eV) :     0.0008022413
SCF cycle converged after 12 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

siesta: E_KS(eV) =             -925.7136

siesta: E_KS - E_eggbox =      -925.7136

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.000000   -0.000000
----------------------------------------
   Max    2.888382
   Res    1.229691    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    2.888382    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):     -160.65      -34.97       -1.15        0.00       -0.00       -0.00
(Free)E + p*V (eV/cell)     -919.5729
Target enthalpy (eV/cell)     -925.7136

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -233.632919
siesta: Eions   =      1621.026762
siesta: Ena     =       347.676395
siesta: Ekin    =       692.878480
siesta: Enl     =      -121.148490
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -2.576473
siesta: DUscf   =         1.551150
siesta: DUext   =         0.000000
siesta: Ex      =      -198.013827
siesta: Ec      =       -25.054094
siesta: Exc     =      -223.067921
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -925.713621
siesta: Etot    =      -925.713620
siesta: FreeEng =      -925.713620

siesta: Final energy (eV):
siesta:  Band Struct. =    -233.632919
siesta:       Kinetic =     692.878480
siesta:       Hartree =     634.973344
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -198.013827
siesta:         Corr. =     -25.054094
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -223.067921
siesta:  Ion-electron =   -1904.211073
siesta:       Ion-ion =    -126.286451
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -925.713620
siesta:         Fermi =      -5.435173

siesta: Atomic forces (eV/Ang):
siesta:      1    0.000000   -2.888382    0.000000
siesta:      2    0.717146    1.455665   -0.000000
siesta:      3   -0.717146    1.455665   -0.000000
siesta:      4    0.000000    2.888382    0.000000
siesta:      5    0.717146   -1.455665   -0.000000
siesta:      6   -0.717146   -1.455665   -0.000000
siesta: ----------------------------------------
siesta:    Tot    0.000000    0.000000   -0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.100271   -0.000000    0.000000
siesta:    -0.000000   -0.021828   -0.000000
siesta:    -0.000000    0.000000   -0.000715

siesta: Cell volume =        150.000000 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00044587          0.00031351  Ry/Bohr**3
siesta:           0.04093816          0.02878515  eV/Ang**3
siesta:          65.59016905         46.11889650  kBar
(Free)E+ p_basis*V_orbitals  =        -924.483567
(Free)Eharris+ p_basis*V_orbitals  =        -924.483567

siesta: Electric dipole (a.u.)  =   -0.000000   -0.000000    0.000000
siesta: Electric dipole (Debye) =   -0.000000   -0.000000    0.000000

cite: Please see "filter_tol.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.402
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.398       1.398   100.00
Setup                  1       0.106       0.106     7.61
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.292       1.292    92.37
geom_init              1       0.629       0.629    44.98
state_init             1       0.049       0.049     3.53
hsparse                1       0.000       0.000     0.03
overlap                1       0.048       0.048     3.47
Setup_H0               1       0.545       0.545    39.00
naefs                  2       0.000       0.000     0.02
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.024       0.049     3.47
nlefsm                 2       0.206       0.411    29.40
DHSCF_Init             1       0.088       0.088     6.26
DHSCF1                 1       0.007       0.007     0.52
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.080       0.080     5.74
REMESH                 1       0.005       0.005     0.33
REORD                 36       0.000       0.004     0.28
PHION                  1       0.062       0.062     4.41
COMM_BSC              59       0.000       0.005     0.32
POISON                15       0.003       0.047     3.34
fft                   30       0.001       0.037     2.65
setup_H               13       0.035       0.451    32.28
DHSCF                 14       0.049       0.690    49.37
DHSCF3                14       0.035       0.487    34.82
rhoofd                14       0.013       0.184    13.14
XC                    14       0.008       0.111     7.92
GXC-CellXC            14       0.008       0.108     7.75
gridxc@cellXC         14       0.008       0.108     7.75
vmat                  14       0.009       0.131     9.40
IterSCF               12       0.035       0.421    30.08
compute_dm            12       0.000       0.002     0.17
diagon                12       0.000       0.002     0.16
r-eigvec              12       0.000       0.002     0.14
r-buildHS             12       0.000       0.000     0.00
rdiag                 12       0.000       0.002     0.13
rdiag1                12       0.000       0.001     0.04
rdiag2                12       0.000       0.000     0.01
rdiag3                12       0.000       0.001     0.07
rdiag4                12       0.000       0.000     0.00
r-buildD              12       0.000       0.000     0.02
MIXER                 11       0.000       0.000     0.01
PostSCF                1       0.242       0.242    17.29
DHSCF4                 1       0.203       0.203    14.55
dfscf                  1       0.190       0.190    13.57
overfsm                1       0.001       0.001     0.04
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.02
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:40:28
Job completed
